pi-manage

```diff
@@ -950,27 +950,32 @@ def create(name, scope, action, filename=None):
 
 
 @api_manager.command
-def createtoken(role=ROLE.ADMIN):
+def createtoken(role=ROLE.ADMIN, days=365, Realm="API", Username=None):
```

Comment:
I think it would be nicer if we could use ``realm`` and ``username`` as variables here. Also because the capitalized ``--Username`` flag looks a bit un-linuxy:
```
  -?, --help            show this help message and exit
...
  -U USERNAME, --Username USERNAME
```
... and I think it's a bit inconsistent because ``pi-manage`` accepts usernames as ``-u`` everywhere else.
 
Maybe we could do that by resorting to [@api_manager.option](https://flask-script.readthedocs.io/en/latest/#adding-options-to-the-manager) here?

Reply:
You can not use ``realm``. The lower case "r" conflicts with the "r" from ``role``.
So I had to do "Realm" (with a capital "R") and so I decided to use a captial "U". :-/

Reply:
Ah, yeah, I think using ``-r``/``--role`` and ``-R``/``--realm`` would be fine. But I guess if we use ``@api_manager.option`` to explicitly define the command-line arguments (like e.g. for ``rotate_audit``), we end up with a little bit of boilerplate, but a nicer command-line interface :)

Reply:
You are totally right...

Reply:
I hope you like it :-)